pkg/virt-launcher/virtwrap/cmd-server/server.go

```diff
@@ -415,6 +415,33 @@ func (l *Launcher) GetDomain(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.D
 
 	return response, nil
 }
+func (l *Launcher) GetQemuVersion(_ context.Context, _ *cmdv1.EmptyRequest) (*cmdv1.QemuVersionResponse, error) {
+
+	response := &cmdv1.QemuVersionResponse{
+		Response: &cmdv1.Response{
+			Success: true,
+		},
+	}
+
+	version, err := l.domainManager.GetQemuVersion()
+
+	if err != nil {
+		response.Response.Success = false
+		response.Response.Message = getErrorMessage(err)
+		return response, nil
+	}
+
+	if jVersion, err := json.Marshal(version); err != nil {
```

Comment:
Move this assignment out of the if/else block to avoid the additional `else` clause?

Reply:
Done!